deepchecks/tabular/dataset.py

```diff
@@ -1061,6 +1065,172 @@ def datasets_share_date(cls, *datasets: 'Dataset') -> bool:
 
         return True
 
+    @property
+    def _dataset_description(self) -> t.Tuple[
+        t.List[t.Union[str, int]],
+        pd.DataFrame,
+        pd.DataFrame
+    ]:
+        data = self.data
+        datetime_name = self.datetime_name
+        index_name = self.index_name
```

Comment:
index_name can't be None?

Reply:
sorry, did not get your question/
but, it can be None

![image](https://user-images.githubusercontent.com/71635444/171125498-cf32cb64-6875-43ed-aae3-6a114abd1b31.png)

Reply:
I'm not sure how you handle the case which index_name is none

Reply:
I just use dataframe index
hm/ that is probably a wrong way/
probably better to remove this `else` branch?

```python
# def __dataset_description(): lines:1109:1113
    else:
        data = data.reset_index()
        index_name = 'index'
        columns = [index_name, *columns]

```